packages/common/src/ui/icons/MSupplyGuy.tsx

```diff
@@ -19,15 +19,15 @@ const sizes = {
 };
 
 type MSupplyGuyProps = SvgIconProps & {
-  size: 'large' | 'medium';
+  size?: 'large' | 'medium';
 };
 
 const SvgGuy: FC<SvgIconProps & { fill: string }> = props => {
   const { fill, ...svgProps } = props;
   return (
     <SvgIcon {...svgProps} viewBox="3 1 206 310">
       <g>
-        <linearGradient id="6000022348f0" x1="0.5" y1="0" x2="0.5" y2="1">
+        <linearGradient id="linear-grad" x1="0.5" y1="0" x2="0.5" y2="1">
```

Comment:
I'm assuming there was no particular reason that this `id` had such an unhelpful name?

Reply:
yeah, laziness 😁